core/controlplane/config/stack_config.go

```diff
@@ -128,35 +55,11 @@ func (c *StackConfig) EtcdSnapshotsS3PrefixRef() (string, error) {
 	return s3path, nil
 }
 
-func (c *StackConfig) ValidateUserData() error {
-	err := userdatavalidation.Execute([]userdatavalidation.Entry{
-		{Name: "UserDataWorker", Content: c.UserDataWorker},
-		{Name: "UserDataController", Content: c.UserDataController},
-		{Name: "UserDataEtcd", Content: c.UserDataEtcd},
-	})
-
-	return err
-}
```

Comment:
I believe we'd better not to remove ValidateUserData but update it to validate against the `s3` and `instance` parts of each UserData like:
```
err := userdatavalidation.Execute([]userdatavalidation.Entry{
  {Name: "UserDataWorker", Content: c.UserDataWorker.Parts.instanec},
  {Name: "UserDataWorker", Content: c.UserDataWorker.Parts.s3},
  {Name: "UserDataController", Content: c.UserDataController.Parts.instance},
  {Name: "UserDataController", Content: c.UserDataController.Parts.s3},
  {Name: "UserDataEtcd", Content: c.UserDataEtcd.Parts.instance},
  {Name: "UserDataEtcd", Content: c.UserDataEtcd.Parts.s3},
})
```